c7n/resources/dynamodb.py

```diff
@@ -450,3 +454,235 @@ class resource_type(object):
         name = 'TableName'
         date = 'CreationDateTime'
         dimension = 'TableName'
+
+
+@resources.register('dax')
+class DynamoDbAccelerator(query.QueryResourceManager):
+
+    class resource_type(object):
+        service = 'dax'
+        type = 'cluster'
+        enum_spec = ('describe_clusters', 'Clusters', None)
+        detail_spec = None
+        id = 'ClusterArn'
+        name = 'ClusterName'
+        config_type = 'AWS::DAX::Cluster'
+        filter_name = None
+        dimension = None
+        date = None
+
+    retry = staticmethod(get_retry(('Throttled',)))
+    filters = FilterRegistry('dynamodb-dax.filters')
+    filters.register('marked-for-op', TagActionFilter)
+    filter_registry = filters
+    permissions = ('dax:ListTags',)
+
+    def get_source(self, source_type):
+        if source_type == 'describe':
+            return DescribeDaxCluster(self)
+        elif source_type == 'config':
+            return query.ConfigSource(self)
+        raise ValueError('invalid source %s' % source_type)
+
+
+class DescribeDaxCluster(query.DescribeSource):
+
+    def augment(self, clusters):
+        resources = super(DescribeDaxCluster, self).augment(clusters)
+        return list(filter(None, _dax_cluster_tags(
+            resources,
+            self.manager.session_factory,
+            self.manager.executor_factory,
+            self.manager.retry,
+            self.manager.log)))
+
+
+def _dax_cluster_tags(tables, session_factory, executor_factory, retry, log):
+
+    def process_tags(r):
+        client = local_session(session_factory).client('dax')
+        tags = []
+        try:
+            tags = retry(
+                client.list_tags, ResourceName=r['ClusterArn'])['Tags']
+        except ClientError as e:
+            if e.response['Error']['Code'] in (
+                    'ClusterNotFoundFault',
+                    'InvalidARNFault',
+                    'InvalidClusterStateFault',
+                    'InvalidParameterCombinationException',
+                    'InvalidParameterValueException'):
+                log.warning(
+                    'Exception collecting tags for %s: \n%s' % (
+                        r['ClusterName'], e))
```

Comment:
missing a continue and raise, to pass through other errors, which are otherwise silently passing here.